src/current/_data/releases.yml

```diff
@@ -5217,3 +5217,31 @@
     docker_arm_limited_access: false
   source: true
   previous_release: v23.2.0-alpha.7
+
+- release_name: v23.1.13
+  cloud_only: true
```

Comment:
ah so this is the new YAML key

would we also need to backfill this key for previous releases? my sense is no

Reply:
You can actually just remove all the `cloud_first*` fields from the release's entry in `releases.yml` and the download links will magically appear.